db/migrate/20230503143849_create_likes.rb

```diff
@@ -0,0 +1,11 @@
+class CreateLikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+      t.references :post, foreign_key: true
+      t.references :author, foreign_key: { to_table: 'users' }
+      t.references :post, foreign_key: true
```

Comment:
- [ Required ] Good job fixing some of the changes required by the previous reviewer, however, I noticed that one migration in `20230503143849_create_likes.rb` fails. I think this is because this line `t.references :post, foreign_key: true` of code has been repeated two times on lines `4` and `6`. Please consider refactoring this migration as shown below to fix the error.

```ruby
class CreateLikes < ActiveRecord::Migration[7.0]
  def change
    create_table :likes do |t|
      t.references :post, foreign_key: true
      t.references :author, foreign_key: { to_table: 'users' }
      t.timestamps
    end
  end
end
```

## Project requirements
![Screenshot from 2023-05-03 22-29-24](https://user-images.githubusercontent.com/16392046/236028571-9f858f2e-6259-499d-9328-b626c27a6dbc.png)

Reply:
@Felix45  Please, is their any other changes? am confused on how many changes each reviewer is coming up with each time i ask for a review

Reply:
I think if that was the only change, you should not block for an aproval

Reply:
Hi @AbdimulhinYussuf3675
I requested this change because the migrations were not working as a result of changes made to the migration files, this means some of the requirements were not met.